### ISSUE TITLE
Attribute of Geod instance changes

### ISSUE THREAD

**Issue by bartvle** [jswhit/pyproj#119]
With pyproj 1.9.5.1 on Python 3.5 on a Windows 10 machine I get

```
>>> import pyproj

>>> g1 = pyproj.Geod(ellps='sphere')
>>> print(g1.a)
6370997.0
>>> g2 = pyproj.Geod(ellps='WGS84')
>>> print(g2.a)
6378137.0
>>> print(g1.a)
6378137.0
```

The 'a' attribute' of 'g1' changes. Is this the expected behavior?

**Comment by micahcochran** [comment]
This was fixed in the repository (issue #64, PR #65).

**Comment by bartvle** [comment]
Thank you! Will there be a new release with this fix?

**Comment by micahcochran** [comment]
I think that fix went into version 1.9.6.
